app/actions/room.ts

```diff
@@ -0,0 +1,108 @@
+import { Action } from 'redux';
+
+import { ROOM } from './actionsTypes';
+
+// TYPE RETURN RELATED
+type ISelected = Record<string, string>;
+
+export interface ITransferData {
+	roomId: string;
+	userId?: string;
+	departmentId?: string;
+}
+
+// ACTION RETURN RELATED
+interface IBaseReturn extends Action {
+	rid: string;
+}
+
+type TSubscribeRoom = IBaseReturn;
+type TUnsubscribeRoom = IBaseReturn;
+type TCloseRoom = IBaseReturn;
+
+type TRoom = Record<string, any>;
+
+interface ILeaveRoom extends Action {
+	roomType: string;
```

Comment:
I think it could be a type of `'channel' | 'team'`

Reply:
in fact I saw that there are several of these types, but there is no Enum, I created one.
Check if that's the case.